docs/source/tutorials/outliers.ipynb

```diff
@@ -528,9 +651,13 @@
     "# Note: This cell is only for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.\n",
     "\n",
     "# Verify the top identified test outliers data are mostly non-animal images\n",
-    "num_animals = len([i for i in range(len(top_outlier_subset)) if top_outlier_subset[i][1] in animal_classes])\n",
-    "if 1 - (num_animals / len(top_outlier_subset)) < 0.81:\n",
-    "    raise Exception(\"Not enough non-animal images amongst top-ranked outliers in test_data.\")"
+    "num_animals = len([i for i in range(len(top_ood_features_subset)) if top_ood_features_subset[i][1] in animal_classes])\n",
+    "if 1 - (num_animals / len(top_ood_features_subset)) < 0.81:\n",
+    "    raise Exception(\"Not enough non-animal images amongst top-ranked outliers in test_data.\")\n",
+    "    \n",
+    "num_animals = len([i for i in range(len(top_ood_predictions_subset)) if top_ood_predictions_subset[i][1] in animal_classes])\n",
+    "if 1 - (num_animals / len(top_ood_predictions_subset)) < 0.51:\n",
```

Comment:
we should trying tweaking settings of the model until this 0.51 is at least 0.65 or so, to hopefully ensure somewhat reasonable performance. Could also try non-LogisticRegression sklearn models such as  `KNeighborsClassifier(n_neighbors=20, weights='distance', metric='cosine')` if just altering the LogisticRegression hyperparameters does not suffice to get good performance.

Reply:
Similar performance to logistic regression. Visually, all the examples look somewhat like outliers so not sure if 0.51 vs 0.65 (1 example) make a big difference.

Reply:
I just think the evaluation is not great if we're only getting 0.51 (essentially random performance under our metric) yet we think the results are pretty good. The fact that we've decided to evaluate via animal vs non-animal means we believe this distinction to be more meaningful than finding strange animal images. If not, we should reconsider the setup I think. But I like this setup of having evaluation where true OOD examples = those whose class was not present in the training data.